include/mrdox/Support/Error.hpp

```diff
@@ -39,7 +39,7 @@ struct Located
 
     template<class Arg>
     requires std::is_constructible_v<T, Arg>
-    Located(
+    consteval Located(
```

Comment:
fuuuuck... I was using `constexpr`? Is that no good anymore? wtf is `consteval` on a function?